### ISSUE TITLE
Memory Editor address textbox doesn't always update

### ISSUE THREAD

**Issue by ioev** [grumpycoders/pcsx-redux#803]
I'm sorry if this has been brought up before, but when clicking to select memory locations in the Memory Editor, often the textbox in the Options bar at the bottom doesn't update.

When clicking an address and the window is out of focus, it always updates.

Afterwards clicking addresses will sometimes not update, or updates with the previously clicked address.

**Comment by nicolasnoble** [comment]
Right, there are several small UX issues with the memory editor. It's currently an external component from https://github.com/ocornut/imgui_club but I might have to make a local copy to incorporate fixes and improvements.
